Doc/library/dis.rst

```diff
@@ -280,6 +287,7 @@ details of bytecode instructions as :class:`Instruction` instances:
 
 
 The Python compiler currently generates the following bytecode instructions.
+Every instruction takes 2 bytes.
```

Comment:
I think this should be moved to the top of the document, under impl-detail block, and decorated with versionchanged.